root/backend/server/server.js

```diff
@@ -26,6 +27,16 @@ db.on('error', console.error.bind(console, 'mongo connection error'));
 
 app.use(router);
 
+app.delete('/admin/:pollUuid', async (req, res, next) => {
+  try {
+    await Poll.findByIdAndDelete({ _id: req.params.pollUuid }).then(data => data);
```

Comment:
what is the purpose of the `.then()` usage here?

Reply:
No purpose